app/utils/pauseBreaksShortcut.js

```diff
@@ -0,0 +1,44 @@
+const { UntilMorning } = require('./untilMorning')
```

Comment:
Could you please what do you think about registering `pauseBreaksToggleShortcut`? Haven't had time think myself, but it hits me we have one shortcut left in `main.js` regarding to pausing.

Or even put all shortcuts here? (like skipToNextScheduledBreakShortcut, skipToNextMiniBreakShortcut and such).

Reply:
Done